get-started/codelab/index.md

```diff
@@ -138,18 +137,18 @@ where the Dart code lives.
   of Material widgets.
 - The main method specifies fat arrow (`=>`) notation,
   which is short hand used for one-line functions or methods.
-- The app extends StatelessWidget which makes the app itself a
+- The app extends `StatelessWidget` which makes the app itself a
```

Comment:
We usually don't codefont class names, but it IS Google style guidelines, so it's fine.